### DISCUSSION TITLE
how can i configure DPI output of the image?

### DISCUSSION THREAD

**Discussion by pipe878** [lllyasviel/Fooocus#1295] [Q&A]
Hi, I'm very new at this, and i'm using advanced properties with high quality.
is there a way to choose the DPI  of the image?
is there a way to select a custom ratio that's not in the displayed options?

Thanks !!

**Comment by ostap667inbox** [comment]
1. The size of the image is specified in pixels. DPI does not refer to bitmap images, only to the devices or surfaces on which the image is displayed. DPI refers to how many pixels of an image fit into 1 inch.
2. You can create a config.txt file or edit an existing one in the Fooocus folder. There, among other settings, you can specify all the necessary proportions for the images that will be displayed in the WebUI. In the same folder there is a file config_modification_tutorial.txt. In it there are examples of settings, study them before writing your own settings.
